### ISSUE TITLE
integrate install doesn't work any more

### ISSUE THREAD

**Issue by dannypike** [microsoft/vcpkg#37467]
It used to work, now it doesn't. It does not prompt for admin permission and it does not set the toolchain file, so that CMake fails, even on a brand new project that attempts to reference only a single vcpkg project.

It claims to have succeeded, but it hasn't actually done so. This is very poor programming style. If you claim that it has worked, you should verify that first.

**Comment by Neumann-A** [comment]
As far as i remember vcpkg integrate install is for MSBuild not CMake

**Comment by dannypike** [comment]
Thanks for your response.

Sorry, I was a bit unclear. I have been using vcpkg for years now and it's worked fine to use with CMake projects in Visual Studio.

Now, it doesn't seem to work. There is no toolchain file passed on the command line that Visual Studio uses to include CMake in a CMake-based project.

There's absolutely no feedback from vcpkg as to why it (a) doesn't try to swap to Administrator mode (which it always used to) and (b) doesn't seem to update Visual Studio' settings, so that I can "find_package()" a vcpkg project such as one or more of the "boost" components.

I tried creating a "blank" CMake project with Visual Studio and the generation phase fails because CMake cannot find the package that I have installed with vcpkg. When I look at the Visual Studio output, it is clear that the toolchain file is not being passed on the command line.

Running `vcpkg integrate install` is quite happy to claim that it worked:

```
D:\work\github\vcpkg>vcpkg integrate install
Applied user-wide integration for this vcpkg root.
CMake projects should use: "-DCMAKE_TOOLCHAIN_FILE=D:/work/github/vcpkg/scripts/buildsystems/vcpkg.cmake"

All MSBuild C++ projects can now #include any installed libraries. Linking will be handled automatically. Installing new libraries will make them instantly available.
```
